### ISSUE TITLE
Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. if custom ID used for Description component

### ISSUE THREAD

**Issue by NuclleaR** [radix-ui/primitives#3579]
### Current Behavior

If custom id used for `Description` component warning appears in the console

```
<Description id="my-id" className="text-md mt-4 text-center font-body text-accent-d" data-role="description">
  {description}
</Description>
```

### Expected behavior

No warning in the console

### Reproducible example

https://codesandbox.io/p/devbox/adoring-jennings-hnzvm9?workspaceId=ws_RwqhyAvK5BpEaJ7RcGktvy

**Comment by nayounsang** [comment]
Can I try this issue?

**Comment by nayounsang** [comment]
In the related internal logic, the difference occurs as `{ descriptionId: 'radix-«rn»', describedById: 'custom-description' }`. I gave a custom ID called `custom-description` of Description.
Find the element corresponding to descriptionId within the DOM and check its accessibility.

**Comment by nayounsang** [comment]
It's a bit of a cautionary tale, but supporting the ability to directly specify ids seems difficult because it would require a lot of codebase changes. Since you can access elements using refs when using React, team'll probably want to avoid specifying ids directly by specifying typescript types, or have users set aria to null themselves (like there's no description).
I think the former is correct, because the title would have the same problem.
